msgid ""
msgstr ""
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2023-03-07 13:51+0000\n"
"Language: en-US\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"

#: apps/auth/views.py:47
msgid "The authentication data is from more than a day ago."
msgstr ""

#: apps/auth/views.py:50
msgid "The authentication data is not related to the Telegram user!"
msgstr ""

#: apps/tdlib/views.py:28
msgid "Invalid offset."
msgstr ""

#: apps/tdlib/views.py:32
msgid "Invalid limit."
msgstr ""

#: apps/tdlib/views.py:36 apps/tdlib/views.py:98 apps/tdlib/views.py:156
#: apps/tdlib/views.py:232 apps/tdlib/views.py:345
msgid "Invalid parent id."
msgstr ""

#: apps/tdlib/views.py:53 apps/tdlib/views.py:78 apps/tdlib/views.py:114
#: apps/tdlib/views.py:175 apps/tdlib/views.py:251 apps/tdlib/views.py:349
msgid "Parent folder not found."
msgstr ""

#: apps/tdlib/views.py:95 apps/tdlib/views.py:153
msgid "Invalid folder name."
msgstr ""

#: apps/tdlib/views.py:134 apps/tdlib/views.py:150 apps/tdlib/views.py:194
msgid "Invalid folder id."
msgstr ""

#: apps/tdlib/views.py:138 apps/tdlib/views.py:169 apps/tdlib/views.py:203
msgid "Folder not found."
msgstr ""

#: apps/tdlib/views.py:189 apps/tdlib/views.py:265
msgid "Invalid delete_from_telegram_chat."
msgstr ""

#: apps/tdlib/views.py:226 apps/tdlib/views.py:270 apps/tdlib/views.py:369
msgid "Invalid file id."
msgstr ""

#: apps/tdlib/views.py:229 apps/tdlib/views.py:352 apps/tdlib/views.py:382
msgid "Invalid file name."
msgstr ""

#: apps/tdlib/views.py:245 apps/tdlib/views.py:279 apps/tdlib/views.py:374
#: apps/tdlib/views.py:415
msgid "File not found."
msgstr ""

#: apps/tdlib/views.py:295
msgid "Invalid file part."
msgstr ""

#: apps/tdlib/views.py:300
msgid "Invalid file object."
msgstr ""

#: apps/tdlib/views.py:305
msgid "Invalid total parts."
msgstr ""

#: apps/tdlib/views.py:310 apps/tdlib/views.py:324
msgid "Invalid file size."
msgstr ""

#: apps/tdlib/views.py:313
msgid "The file size must be less or equal to 2 GB."
msgstr ""

#: apps/tdlib/views.py:317
msgid "The file part size must be less or equal to 512 KiB."
msgstr ""

#: apps/tdlib/views.py:321
msgid "The file part size must be evenly divisible by 1024."
msgstr ""

#: apps/tdlib/views.py:329
msgid "total_parts must be {}."
msgstr ""

#: apps/tdlib/views.py:334
msgid "Provided file_size is not equal to the actual file size."
msgstr ""

#: apps/tdlib/views.py:339
msgid "Invalid md5 checksum."
msgstr ""

#: apps/tdlib/views.py:378
msgid "File with id {} successfully uploaded before."
msgstr ""

#: apps/tdlib/views.py:386
msgid "You must upload part {} now."
msgstr ""

#: apps/tdlib/views.py:391
msgid "The file part size must be equal to {} KiB."
msgstr ""

#: apps/tdlib/views.py:396
msgid "The file part size must be less or equal to {} KiB."
msgstr ""

#: apps/tdlib/views.py:434
msgid "Invalid Range header."
msgstr ""

#: baseapi/settings.py:21
msgid "English"
msgstr ""
